flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCOutputFormat.java

```diff
@@ -207,13 +238,18 @@ public void writeRecord(Row row) throws IOException {
 
 		if (batchCount >= batchInterval) {
 			// execute batch
+			batchLimitReachedMeter.markEvent();
```

Comment:
this seems redundant given that `flushMeter` exists. While the job is running `batchLimit == flushMeter`, and at the end `batchLimit == flushMeter -1` except in the exceedingly rare case that the total number of rows fits perfectly into the batches.